vulcan/lib/server/workflows/scRNAseq.cwl

```diff
@@ -321,6 +321,7 @@ steps:
   pick_DGE_methods:
     run: ui-queries/diff-exp-sc.cwl
     label: 'Set how DE should be run'
+    doc: 'Sets the cells compared in differential gene expression calculations.  First, (step 1) you will select your question type, and then (step 2, optional) if there are certain cells you wish to exclude, click the subset checkbox and select the cells which you with to KEEP.  Afterwards, the exact needs (step 3, and sometimes also step 4) will depend on your question type selection of step one, but the overall idea is that these steps fill in the specific sets of cells to compare. See the dedicated section of the overall vignette for more detail. Also note: Sections 7 & 8 of the primary inputs contain additional inputs which contribute to the (7) calculation method or (8) post-calculation filtration cutoffs.'
```

Comment:
(nits): I found the following typos:

`For users initerested in adjusting these parameters` should be `For users interested in adjusting these parameters`

`Selecting of which single-cell data` should be `Selecting which single-cell data`?

`performed to idenfiy top markers` should be `performed to identify top markers`

`versus every other cluster in. These` ... seems to be missing something at the end of the sentence? Not sure what the words should be, though...

`Visulaization and Cluster Annotation (optional)` should be `Visualization and Cluster Annotation (optional)`

`the preliminary differntial expression results` should be `the preliminary differential expression results`

`the final signel-cell data object` should be `the final single-cell data object`

`to perforfollow up analysis outside of vulcan.` should be `to perform follow up analysis outside of Vulcan.` ?

(sorry, can't figure out how to do diffs / comments on specific sections of a single line string...)